lib/versioning/node/index.spec.ts

```diff
@@ -46,14 +59,68 @@ describe('versioning/node/index', () => {
       ${'10.1.0'}   | ${t1} | ${true}
       ${'10.0.0a'}  | ${t1} | ${false}
       ${'9.0.0'}    | ${t1} | ${false}
-    `('isStable("$version") === $expected', ({ version, time, expected }) => {
+    `('("$version") === $expected', ({ version, time, expected }) => {
       DateTime.local = (...args: (string | any)[]) =>
         args.length ? dtLocal.apply(DateTime, args) : time;
-      expect(isStable(version as string)).toBe(expected);
+      expect(nodever.isStable(version as string)).toBe(expected);
```

Comment:
Why do we need to cast `version` here? `version` is `any`
```suggestion
      expect(nodever.isStable(version)).toBe(expected);
```

Reply:
> Why do we need to cast version here? version is any

Strict null and implicit any checks are already enabled for node versioning

Reply:
What @zharinov says :) I don't know if there is a better way @viceice?